api/lanes.py

```diff
@@ -608,26 +529,19 @@ class ContributorLane(QueryGeneratedLane):
     ROUTE = 'contributor'
     MAX_CACHE_AGE = 48*60*60    # 48 hours
 
-    def __init__(self, _db, contributor_name, contributor_id=None,
```

Comment:
Is it good to start using `contributor_id` now? Or should we wait until it's more likely that two contributors with the same name are different people? I feel like right now there are a lot of duplicate contributors, and searching by name is more reliable than searching by ID.